service/runner.cc

```diff
@@ -90,9 +90,14 @@ CreateStdPipe(bool forWriting)
 
 } // namespace
 
+
 namespace Datacratic {
 
-/* ASYNCRUNNER */
+/****************************************************************************/
+/* RUNNER                                                                   */
+/****************************************************************************/
+
+std::string Runner::runnerHelper;
```

Comment:
If I understand correctly, this var is only there for testing? It should be indicated.